src/pipeline.py

```diff
@@ -8,32 +8,35 @@
         os.path.abspath(__file__)))
 
 
-def data_standardization(config, DATA_FP, forceupdate=False):
+def data_standardization(config_file, DATA_FP, forceupdate=False):
     """
     Standardize data from a csv file into compatible crashes and concerns
     according to a config file
     Args:
-        config
+        config_file
         DATA_FP - data directory for this city
     """
     # standardize data, if the files don't already exist
     # or forceupdate
-    print("Standardizing data...")
     if not os.path.exists(os.path.join(
             DATA_FP, 'standardized', 'crashes.json')) or forceupdate:
+        print("Standardizing crash data..")
         subprocess.check_call([
             'python',
             '-m',
             'data_standardization.standardize_crashes',
+            '-c',
+            config_file,
             '-d',
-            config['name'],
-            '-f',
             DATA_FP
         ])
     else:
         print("Already standardized crash data, skipping")
 
     # There has to be concern data in the config file to try processing it
```

Comment:
Can you just move the comment below the loading of the config file, for clarity? Otherwise looks good!

Reply:
Sorry I'm not following, can you reword your request (or we can chat about it in the next meeting).

Also I haven't yet implemented the `..(end_year is not None and crash_year > (end_year - 1))` change, would doing so be a good way for us to at least be in sync of our use of end_year in the short term?